src/onegov/org/views/ticket.py

```diff
@@ -445,11 +445,11 @@ def email_iter():
     model=Ticket, name='note', permission=Private,
     template='ticket_note_form.pt', form=TicketNoteForm
 )
-def handle_new_note(self, request, form, layout=None):
+def handle_new_note(self: Ticket, request: 'OrgRequest', form: TicketNoteForm,
+                    layout: TicketNoteLayout | None = None):
 
     if form.submitted(request):
-        message = form.text.data,
-        form: TicketNoteForm
+        message = form.text.data
```

Comment:
Since `TicketNoteForm` uses `InputRequired` on `text` you can assert here that the message is not `None`, so this solves our mystery from above and you can change the argument back and instead add:

`assert message is not None`